espresso/core/java/androidx/test/espresso/matcher/ViewMatchers.java

```diff
@@ -757,7 +760,9 @@ private WithTextMatcher(Matcher<String> stringMatcher) {
     @Override
     protected void describeMoreTo(Description description) {
       description.appendText("view.getText() with or without transformation to match: ");
-      stringMatcher.describeTo(description);
+      StringDescription buffer = new StringDescription();
+      stringMatcher.describeTo(buffer);
+      description.appendText(highlightNbsp(buffer.toString()));
```

Comment:
Can you please add test cases?

Reply:
Hi, @adazh 
That's where trouble appeared. I didn't manage to build the project completely, some imports could not be resolved.
Previously, I put tests for this in my project. 
I would add checks if you could help me make it through it.

Reply:
Sure, do you mind pasting the build errors here?

Reply:
Here is sync log https://gist.github.com/Kutashov/3ada8fbe4b4861d40c961b0c3d4612ff.
Env: ubuntu 20.04, bazel 3.5.0, python 2.7

Reply:
Hi @adazh. How're you doing?

I still have no luck in fixing this problem. Would you mind giving me some hints on this?

Reply:
Sorry for the delay! Not sure why but looks like it an issue with building some `kt_jvm_library`:
```
ERROR: /home/akutashov/as-workspace/android-test/espresso/core/java/androidx/test/espresso/device/context/BUILD.bazel:12:19: in kt_jvm_library rule //espresso/core/java/androidx/test/espresso/device/context:context_kt: 
Traceback (most recent call last):
	File "/home/akutashov/as-workspace/android-test/espresso/core/java/androidx/test/espresso/device/context/BUILD.bazel", line 12, column 19, in kt_jvm_library(name = 'context_kt')
		kt_android_library(
	File "/home/akutashov/.cache/bazel/_bazel_akutashov/2b5975f9cd3064037d88fb6d16b027c0/external/io_bazel_rules_kotlin/kotlin/internal/jvm/impl.bzl", line 235, column 36, in kt_jvm_library_impl
		_kt_jvm_produce_jar_actions(ctx, "kt_jvm_library") if ctx.attr.srcs else export_only_providers(
	File "/home/akutashov/.cache/bazel/_bazel_akutashov/2b5975f9cd3064037d88fb6d16b027c0/external/io_bazel_rules_kotlin/kotlin/internal/jvm/compile.bzl", line 591, column 42, in kt_jvm_produce_jar_actions
		source_jar = java_common.pack_sources(
Error in pack_sources: pack_sources() missing 1 required named argument: host_javabase
```

Can you try using bazel 4.2.1? It's the version we are currently using to the the binaries: https://github.com/android/android-test/blob/master/.bazelversion.

Reply:
Ok, this info https://github.com/android/android-test/blob/master/CONTRIBUTING.md#one-time-setup is a bit out of date.

I moved to 4.2.1 but still got some linking errors. Could you please take a look? https://gist.github.com/Kutashov/04b0221f8aa235997a97e2139a276ae9

Reply:
Hi.
Kind reminder: I'm still looking forward to your help :)

Reply:
Hi. I've updated the CONTRIBUTING.md file. I have also fixed a small number of BUILD files to address the issue seen in your gist with the espresso/matcher tests (thanks for the gist output by the way, it helped a lot).
Once you sync head up to 220d95c3403c919b7c6d2ad01b89a2ec88da1d89, you should be able to run the tests specific to your changes, for example:
$ bazel test //espresso/core/javatests/androidx/test/espresso/matcher/... --host_force_python=PY2

Note that I only tried building & running the tests from command-line rather than from IJ, in case it makes a difference for you. Please let me know if you need more to unblock you.

Reply:
Ralf, thanks a lot for your assistance!
Finally, I managed to run test :+1: There are a bunch of errors during project sync though :)